src/components/Stack/Stack.stories.tsx

```diff
@@ -7,159 +7,109 @@ export default {
-      }}
-    />
-    <Box
-      width={24}
-      height={24}
-      style={{
-        background: 'lightgray',
-      }}
-    />
-    <Box
-      width={24}
-      height={24}
+const SPACE_UNIT = 8;
+
+const Item = () => <Box width={24} height={24} style={{ background: '#00E3D8' }} />;
+
+export const RowDefinedGap = () => (
+  <Box>
+    <Stack
+      gap={3 * SPACE_UNIT}
       style={{
-        background: 'lightgray',
+        background: '#EAEAFB',
       }}
-    />
-  </Stack>
+    >
+      <Item />
+      <Item />
+      <Item />
+      <Item />
+    </Stack>
+  </Box>
 );
 
-export const ColumnUnits3 = () => (
-  <Stack
-    column
-    units={3}
-    style={{
-      background: '#f0f0f0',
-    }}
-  >
-    <Box
-      width={24}
-      height={24}
-      style={{
-        background: 'lightgray',
-      }}
-    />
-    <Box
-      width={24}
-      height={24}
-      style={{
-        background: 'lightgray',
-      }}
-    />
-    <Box
-      width={24}
-      height={24}
+export const ColumnDefinedGap = () => (
+  <Box>
+    <Stack
+      column
+      gap={3 * SPACE_UNIT}
       style={{
-        background: 'lightgray',
+        background: '#EAEAFB',
       }}
-    />
-    <Box
-      width={24}
-      height={24}
-      style={{
-        background: 'lightgray',
-      }}
-    />
-  </Stack>
+    >
+      <Item />
+      <Item />
+      <Item />
+      <Item />
+    </Stack>
+  </Box>
 );
 
 export const RowFluid = () => (
   <div
     style={{
       width: '400px',
-      background: '#f0f0f0',
+      background: '#EAEAFB',
     }}
   >
     <Stack fluid grow>
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
+      <Item />
+      <Item />
+      <Item />
+      <Item />
     </Stack>
   </div>
 );
 
 export const ColumnFluid = () => (
-  <div
-    style={{
-      height: '400px',
-      background: '#f0f0f0',
-    }}
-  >
-    <Stack column fluid grow>
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
-      <Box
-        width={24}
-        height={24}
-        style={{
-          background: 'lightgray',
-        }}
-      />
+  <Box>
+    <div
+      style={{
+        height: '400px',
+        background: '#EAEAFB',
```

Comment:
I was wondering if we could consider having colors stored in variables making use of tools such as scss in the future

Reply:
we usually have colors stored, in this case the main purpose of the lib is to esport some layout components and colors would be needed just in the storybook. We have a task in clickup to apply fastloop.it look and feel to the storybook UI we could use that to create a color palette to use in storybook!